z2/src/chain.rs

```diff
@@ -151,6 +152,17 @@ impl Chain {
         }
     }
 
+    pub fn get_contract_upgrades_block_heights(&self) -> ContractUpgradesBlockHeights {
+        match self {
+            Self::Zq2Devnet => ContractUpgradesBlockHeights {
+                deposit_v3: Some(3600),
```

Comment:
is `deposit_v3` expected in devnet to be always at block 3600?

Reply:
In devnet it can be set to 3600 or even less in order to reduce the time we're waiting for the upgrade to happen.

Reply:
The value determines at which height a particular contract upgrade will occur. So assuming no network `reset` the value for a particular contract (`deposit_v3` in this case) will never change. 

However more contract upgrades may be added at later dates, requiring a network `upgrade`.